aws/logs_monitoring/setup.py

```diff
@@ -12,12 +12,12 @@
     author="Datadog, Inc.",
     author_email="dev@datadoghq.com",
     classifiers=[
-        "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
     ],
     keywords="datadog aws lambda layer",
-    python_requires=">=3.7, <3.10",
-    install_requires=["datadog-lambda==3.39.0", "requests-futures==1.0.0"],
+    python_requires=">=3.9, <3.11",
+    install_requires=["datadog-lambda==4.77.0", "requests-futures==1.0.0"],
```

Comment:
Where's that version change coming from ?

Reply:
required dependency version update to support versions 3.10 -> 3.11